ChromaDB.Client/Services/Implementations/ChromaCollectionClient.cs

```diff
@@ -33,4 +33,11 @@ public async Task<BaseResponse<CollectionEntriesQueryResponse>> Query(Collection
 			.Add("{collection_id}", _collection.Id);
 		return await _httpClient.Post<Collection, CollectionQueryRequest, CollectionEntriesQueryResponse>(request, requestParams);
 	}
+
+	public async Task<BaseResponse<BaseResponse.None>> Add(CollectionAddRequest request)
```

Comment:
Am new to this SDK so my question may be totally uninformed, but... What's the policy about exposing types like CollectionAddRequest to users? What I mean is, should APIs expose method parameters instead - at least for mandatory stuff like `Ids` - to make for a nicer .NET API, rather than exposing what is essentially an internal REST JSON DTO?

Reply:
For now I'm following already existing APIs. But the cleanup is on todo https://github.com/ssone95/ChromaDB.Client/issues/27.